brevitas/proxy/runtime_quant.py

```diff
@@ -201,6 +202,7 @@ def __init__(self,
                                               signed=signed,
                                               restrict_scaling_type=restrict_scaling_type)
                 tensor_quant = RescalingIntQuant(signed=signed,
```

Comment:
This goes away

Reply:
How can I remove this, considering the changes [here?](https://github.com/Xilinx/brevitas/pull/128/files/2b72ab0e47a20c8d619d1e0011f4da8068aff64c#diff-479964b2d3ffefe1d9bb8a0d30482df6R493)

Reply:
You are right sorry, I was thinking about Restats